example/lib/03.control/widgets/dropdown_selector.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:flutter/material.dart';
+
+class DropdownSelector extends StatelessWidget {
+  final Iterable<String?> choices;
+  final String value;
+  final IconData icon;
+  final Function(String?)? onChanged;
+
+  const DropdownSelector({
+    super.key,
+    required this.choices,
+    required this.value,
+    required this.icon,
+    required this.onChanged,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return DropdownButton<String>(
+      value: value,
+      items: choices.map((String? value) {
+        return DropdownMenuItem<String>(
+          value: value,
+          child: value == null
+              ? const Divider()
```

Comment:
Does this mean the currently selected item is not visible in the dropdown?

Reply:
No, seems like it doesn't do anything
Removed it.